poetry/console/commands/import_.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from poetry.layouts import layout
+from .command import Command
+from ...utils._compat import Path
+from ...utils.toml_file import TomlFile
+
+
+class ImportCommand(Command):
+    """
+    Create poetry project from existing files (<comment>setup.py</>, <comment>Pipfile</> ...)
+
+    import
+        {--from= : Config file to import}
+        {--package-name= : Name of the package}
+        {--package-version= : Version of the package}
+
+    """
+
+    def __init__(self):
+        super(ImportCommand, self).__init__()
+
+    def handle(self):
+        from_file = self.get_from_file()
+        if not from_file:
+            return 1
```

Comment:
Would it make more sense to return `True / False` rather than 0s and 1s?

Reply:
I think the return value of handle() is the return code of the script, but I may be wrong